50-main.config

```diff
@@ -65,7 +65,7 @@ START = (isUndefined(DaemonStartTime) || (time() - DaemonStartTime) < (MY.ACCEPT
         ($(CPUJOB_ON_GPUSLOT)) && \
         (isUndefined(TARGET.SingularityImage) || MY.SINGULARITY_START_CLAUSE) && \
         (TARGET.Want_MPI =!= True || MY.Has_MPI =?= True) && \
-        ($(START_EXTRA))
+        (START_EXTRA ?: true) && (OSG_PROJECT_RESTRICTION ?: true)
```

Comment:
Where do you advertise START_EXTRA? If START_EXTRA is not an advertised attribute then you need the `$( )` around it to do the macro expansion.

Reply:
`OSG_PROJECT_RESTRICTION` must have the same problem then. Would it be enough to add them to `STARTD_ATTRS`?

Reply:
Adding both to `STARTD_ATTRS` fixed it. Tested and added to this PR.